<launch>
  <master auto="start"/>
  <param name="/use_sim_time" value="false"/>

  <!-- ASIPP Global Planner Node -->
  <node pkg="move_base_3D" type="move_base_3D" name="MoveBase3D" respawn="false" output="screen" clear_params="true">
    <remap from="/map" to="/map" />
    <remap from="/goal_pose" to="/goal_pose" />
    <remap from="/currentPos" to="/currentPos" />
    <rosparam file="$(find sbpl_dynamic_env_global_planner_3D)/config/asipp_params.yaml" command="load"/>
    <param name="primitive_filename" value="$(find sbpl_dynamic_env_global_planner_3D)/config/simple_0.5_diag_heli_time_0.1.mprim" /> 
<!--<param name="primitive_filename" value="$(find sbpl_dynamic_env_global_planner_3D)/config/simple_0.5_diag_heli_time_0.1.mprim" /> --> 
<!--    <param name="primitive_filename" value="$(find sbpl_dynamic_env_global_planner_3D)/config/heli_time_0.1.mprim" /> -->
<!--    <param name="static_collision_map" value="$(find sbpl_dynamic_env_global_planner_3D)/config/wg-room-1.bt" /> -->
<!--    <param name="static_collision_map" value="$(find sbpl_dynamic_env_global_planner_3D)/config/wg_room_1.bt" /> -->
<!--    <param name="static_collision_map" value="$(find sbpl_dynamic_env_global_planner_3D)/config/freiburg1_360.bt" /> -->
    <param name="static_collision_map" value="$(find sbpl_dynamic_env_global_planner_3D)/config/ccny_3rooms_octomap_5cm.bt" />
  </node>

  <!-- Goal and Start interactive markers -->
  <node pkg="goal_interactive_marker" name="goal_interactive_marker" type="goal_interactive_marker">
     <param name="goal_init_pose_x" value="-5.624168" />
     <param name="goal_init_pose_y" value="0.556141" />
     <param name="goal_init_pose_z" value="0.062963" />
     
     <param name="start_init_pose_x" value="5.66191" />
     <param name="start_init_pose_y" value="-1.07912" />
     <param name="start_init_pose_z" value="0.468205" />
  </node>

</launch>

